mithril-common/src/protocol/multi_signer.rs

```diff
@@ -16,6 +17,14 @@ pub struct MultiSigner {
     protocol_parameters: StmParameters,
 }
 
+/// [SignerBuilder] specific errors
+#[derive(Debug, Error)]
+pub enum MultiSignerError {
+    /// Error raised when a single signature can not be verified
+    #[error("Invalid signature for party: '{party_id}'")]
+    InvalidSignature { party_id: String },
```

Comment:
We probably need to embed the source error in a `StdError` as it is a valuable information

Reply:
Not useful anymore with the latest commit.